.github/workflows/ci.yml

```diff
@@ -88,7 +88,7 @@ jobs:
 
   build:
     needs: test
-    if: github.ref == 'refs/heads/master'
+    if: github.ref == 'refs/heads/master' && (github.repository_owner == 'ia-toki' || vars.JUDGELS_IMAGE_AUTOBUILD == 'true')
```

Comment:
Sorry what is `JUDGELS_IMAGE_AUTOBUILD`?

Reply:
It's a repo-level variable for github action, by default this variable is not defined, so the condition is false on fork repo

This logic also applies for `JUDGELS_STAGING_DEPLOYMENT` variable below

Reply:
I don't understand; this `ia-toki/judgels` repository does not have `JUDGELS_IMAGE_AUTOBUILD` or `JUDGELS_STAGING_DEPLOYMENT` variable.

<img width="1133" alt="image" src="https://github.com/ia-toki/judgels/assets/1525580/6daca9ea-e78a-4a6a-9745-b05e8d643e5f">